internal/cache/metrics.go

```diff
@@ -0,0 +1,30 @@
+package cache
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const (
+	metricsPrefix            = "opa_cache_"
```

Comment:
I maybe be wise to use the full repo name here to avoid conflicts in case other sidecar project in this org introduce similar metrics (e.g. `opa-ams`). WDYT?
```suggestion
	metricsPrefix            = "opa_openshift_cache_"
```